### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by hotzenklotz**
Added standard Github issues templates for `Bugs` and `Feature Requests`. Requested by @MichaelBuessemeyer  and @philippotto  

See https://github.com/scalableminds/webknossos/pull/5078

### CODE REVIEW

**Review by philippotto — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (normanrz) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]:
```suggestion
labels: bug
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (normanrz) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]:
```suggestion
labels: enhancement
```

**Review by normanrz — CHANGES_REQUESTED** [review]

**Comment by hotzenklotz** [comment]
@MichaelBuessemeyer or @philippotto Feel free to take over the PR and add the relevant issue templates content.

**Comment by MichaelBuessemeyer** [comment]
I just updated the templates to the content we had before. If everyone is happy we can merge this PR :)

**Review by normanrz — APPROVED** [review]

**Comment by philippotto** [comment]
:+1:

**Pull request merged**